### DISCUSSION TITLE
p-editor (quill)  - Mutation events will be removed

### DISCUSSION THREAD

**Discussion by stephane-englebert** [primefaces/community#1108] [PrimeNG] [answered]
Hello,

I'm using the p-editor component in my new project and everything is working fine.
I just got the following warning in Chrome console :

>  _[Deprecation] Listener added for a synchronous 'DOMNodeInserted' DOM Mutation Event. This event type is deprecated (https://w3c.github.io/uievents/#legacy-event-types) and work is underway to remove it from this browser. Usage of this event listener will cause performance issues today, and represents a risk of future incompatibility. Consider using MutationObserver instead._

and the following one in Powershell console :

> Severity: moderate
> Cross-site Scripting in quill - https://github.com/advisories/GHSA-4943-9vgg-gr5r
> No fix available
> node_modules/quill

I read that [Mutation events will be removed from Chrome](https://developer.chrome.com/blog/mutation-events-deprecation?hl=en) end of july 2024.
**What'll be the impact on p-editor's use?  Will it be still possible to use it?**
I would like to know if I should immediately look for another solution for my project.
Thanks for your feedback!

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We cannot interfere with how the quill works but I believe that they will prepare something as a replacement. We can take action when they do something about it.

**Reply by stephane-englebert** [reply]
Hi,
Oki, i hope so, it's a nice feature.
Thanks for your quick answer!

**Reply by ThoSap** [reply]
As you can see Quill is again under active development since September 2023.
The new v2.0 version, which will be released soon no longer has the mutation events.
https://github.com/quilljs/quill/issues/3806#issuecomment-1933250524
https://github.com/quilljs/quill/releases

**Reply by stephane-englebert** [reply]
Thank you very much for sharing this info!
Have a nice day

**Reply by ThoSap** [reply]
See https://github.com/primefaces/primeng/issues/14721

**Reply by ThoSap** [reply]
@stephane-englebert this is fixed with 🥳🥳🥳
https://github.com/primefaces/primeng/releases/tag/17.18.3

Just update to this version and Quill 2.0.2
